.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # 3.8, 3.9 do not work at the moment.
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v2
+      -
+        name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      -
+        name: Display system info
+        run: |
+          python -c "import sys; print(sys.version)"
```

Comment:
Is there a benefit to replacing this whole line with `python --version`? It shows less information than the way you've got it currently. Is overhead a concern?

Reply:
unsure... it is what is in the GitHub workflow template.